### PR TITLE
Mini-bugfixes + ShaderGUI.

### PR DESCRIPTION
**Pull request by OlePLarsen**
Updated WASD-controller to use deltatime + be adjustable in-scene.
Fixed error with airtapping on a button that would get disabled.

### CODE REVIEW

**Review by Adrian-Wennberg — COMMENTED** [review]
Do we even support the old input system? If we don't I think we should just remove the #if for it and just support the new one.

**Comment by OlePLarsen** [comment]
Eh, sure. Might as well. 

The old one will be kicking around in the commits if anyone ever has a need for it I suppose.

File: Scripts/Runtime/Foundations/Functionallity/Hands/InteractableButton.cs [hunk 1]
```diff
@@ -159,7 +159,8 @@ private void TryPressButton()
             {
                 if (!currentlyAnimating)
                 {
-                    StartCoroutine(PressButton());
+                    //StartCoroutine(PressButton());
+                    PressButton();
```
[the remark this answers is not in the document]
Comment (OlePLarsen) [reply Scripts/Runtime/Foundations/Functionallity/Hands/InteractableButton.cs]: Corrected, now works as expected and still keeps the coroutine: 6d4c208c38452886261b073100aa63515da1da40

**Review by Adrian-Wennberg — DISMISSED** [review]
Looks good!

**Comment by Adrian-Wennberg** [comment]
Could you actually also bump the version number to 1.0.4?

**Review by Adrian-Wennberg — APPROVED** [review]

**Pull request merged**